pkg/apis/rbac/cast.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbac
+
+// Casting utilities to and from "Cluster" level equivalents.
```

Comment:
> We have these to support virtual storage. Without virtual storage, you may not need them

Ok, I see, you picked them up with validation.  I'm ok with them, but try to contain them to the package you're using them in?  I still hold out hope of removing them.